developers/blobstream.md

```diff
@@ -14,89 +14,111 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
 developers to build high-throughput L2s using Celestia's optimised DA layer,
 the first with Data Availability Sampling (DAS).
 
-The following docs go over how developers can integrate Blobstream.
-There are also docs on how to run a [Blobstream orchestrator](../nodes/blobstream-binary.md)
-as a Celestia validator which won't be covered in the following sections
-aimed at developers.
+An implementation of Blobstream, by [Succinct](https://succinct.xyz/), called
+[Blobstream X](https://github.com/succinctlabs/blobstreamx), is out
+and will be used in our canonical deployments. This latter proves Celestia 
```

Comment:
"nit: "This implementation proves validity of Celestia block headers on a target EVM chain using zero-knowledge proofs (ZKPs), which allows inheriting all the security guarantees of Celestia."

Reply:
[d7d2f14](https://github.com/celestiaorg/docs/pull/1379/commits/d7d2f14e872ee4fcc5a06079a3e5b0c29840e0f2)